dashboards/aws-apprunner/aws-apprunner.json

```diff
@@ -0,0 +1,208 @@
+{
+  "name": "AWS AppRunner Dashboard",
+  "description": null,
+  "pages": [
+    {
+      "name": "AWS AppRunner Dashboard",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Application logs",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.line"
+          },
+          "rawConfiguration": {
+            "nrqlQueries": [
+              {
+                "accountId": 0,
+                "query": "SELECT count(*) FROM Log WHERE `aws.logGroup` = '/aws/apprunner/${AppRunner-container-name}/${application-hash}/application' SINCE 2 hours ago facet `aws.logGroup` TIMESERIES MAX"
```

Comment:
I noticed these `${}` templates in a few of the NRQL queries, quickstart dashboards do not currently support any type of templating. So I just want to make sure that the full name of the `aws.logGroup` is literally `/aws/apprunner/${AppRunner-container-name}/${application-hash}/application`. This comment applies to a few on the NRQL queries